dask/dataframe/core.py

```diff
@@ -1560,12 +1624,42 @@ def describe(self, split_every=False, percentiles=None, percentiles_method='defa
                  num.max(split_every=split_every)]
         stats_names = [(s._name, 0) for s in stats]
 
-        name = 'describe--' + tokenize(self, split_every)
-        layer = {(name, 0): (methods.describe_aggregate, stats_names)}
+        colname = data._meta.name if isinstance(data._meta, pd.Series) else None
+
+        name = 'describe-numeric--' + tokenize(num, split_every)
+        layer = {(name, 0): (methods.describe_numeric_aggregate, stats_names, colname, is_timedelta_column)}
         graph = HighLevelGraph.from_collections(name, layer, dependencies=stats)
         meta = num._meta.describe()
         return new_dd_object(graph, name, meta, divisions=[None, None])
 
+    def _describe_categorical_1d(self, data, split_every=False):
```

Comment:
Since this isn't just for categorical dtype series (also does bools, objects, datetimes, and others?), perhaps this method (and the key/task names created by it) should use a different word? `nonnumeric` perhaps?